code/modules/jobs/job_types/roguetown/courtier/jester.dm

```diff
@@ -53,6 +53,7 @@
 			H.mind.AddSpell(new /obj/effect/proc_holder/spell/invoked/projectile/splash)
 			H.mind.AddSpell(new /obj/effect/proc_holder/spell/targeted/roustame)
 			H.mind.AddSpell(new /obj/effect/proc_holder/spell/invoked/projectile/animate_object)
+			H.mind.AddSpell(new /obj/effect/proc_holder/spell/invoked/push_spell)
```

Comment:
nooo don't add it to jester! Jesters have -5 PQ req

Reply:
ok